main/station-config.js

```diff
@@ -124,11 +125,67 @@ function getStationWalletBalance () {
   return 0 // todo - backend logic
 }
 
+function firstFunction () {
```

Comment:
what does `firstFunction` mean?

Reply:
It's a draft. I'll let you know when it's ready for review